subworkflows/nf-core/fastq_fastqc_umitools_trimgalore/tests/main.nf.test.snap

```diff
@@ -152,28 +152,7 @@
             "nf-test": "0.8.4",
             "nextflow": "24.01.0"
         },
-        "timestamp": "2024-02-29T14:41:05.551084"
-    },
-    "pe skip all - reads": {
-        "content": [
-            [
-                [
-                    {
-                        "id": "test",
-                        "single_end": false
-                    },
-                    [
-                        "/nf-core/test-datasets/modules/data/genomics/sarscov2/illumina/fastq/test_1.fastq.gz",
-                        "/nf-core/test-datasets/modules/data/genomics/sarscov2/illumina/fastq/test_2.fastq.gz"
```

Comment:
ok, so I discovered the hard way that this was a failure...
I should have pay more attention to what you guys were saying @sateeshperi @matthdsm